pytest/tests/sanity/block_production.py

```diff
@@ -0,0 +1,58 @@
+# Spins up four nodes, and waits until they produce 100 blocks.
+# Ensures that the nodes remained in sync throughout the process
+# Sets epoch length to 10
+
+import sys, time
+
+sys.path.append('lib')
```

Comment:
why not from .cluster import start_cluster instead of adding lib to path?

Reply:
Because it doesn't seem to work.
Interweb suggests that using a library from a sibling folder is intentionally not supported.